Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM alpine:3.2
+
+RUN apk add --update \
+    php-fpm \
+    php-mcrypt \
+    php-curl \
+    php-openssl \
+    php-phar \
+    php-ctype \
+    php-json \
+    curl \
+    git \
+    php-dom \
+    alpine-sdk \
+    php-dev \
+    autoconf \
+    openssl-dev \
+    php-pdo \
+    php-pdo_pgsql \
+    php-pdo_odbc \
+    php-pdo_mysql \
+    php-pdo_sqlite \
+    php-opcache && \
+    sed -i 's/\;date\.timezone\ \=/date\.timezone\ \=\ Europe\/Berlin/g' /etc/php/php.ini && \
```

Comment:
Why this timezone? Can we make it configurable? 
@Kapeli do we care about timezone at all? 
If not can we make it UTC?